data/digests/34873b4d2390.json

```diff
@@ -0,0 +1,105 @@
+{
+    "id": "34873b4d2390",
+    "title": "Holding the focus",
+    "impactStatement": "A new open source software tool makes it possible for researchers to view cells in living animals in high detail.",
+    "stage": "preview",
+    "updated": "2018-07-30T14:01:23Z",
```

Comment:
If this is input data `updated` does not need to be provided as its now handled by the digest service. It 'can' be passed but will be ignored.

Reply:
This is output, being returned by the API

Reply:
`updated` should depend on `published`.

Reply:
In the data or in the code? I haven't found clashes between the two fields in `bootstrap.php` if not for the `use-date` parameter.

The other two digests here have an `updated` that is greater than `published`, which is consistent. This `preview` digest doesn't have a `published` but it could in principle have one if it is prepopulated (in reality coming from the XML or other similar source). Could add it, but probably we should have two separate cases, one `preview` with it and one `preview` without?

Reply:
In the schema (like https://github.com/elifesciences/api-raml/blob/817a5e24bd5762ec84147cee09bbdbbf6b1a83fc/src/blocks/box.v1.yaml#L31-L32).

Sounds like we do need a third stage if there's a `published` date but its not yet published.